FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/RobotEncoder.java

```diff
@@ -0,0 +1,29 @@
+package org.usfirst.frc.team1076.robot.physical;
+
+import org.usfirst.frc.team1076.robot.sensors.IEncoder;
+import edu.wpi.first.wpilibj.Encoder;
+
+public class RobotEncoder implements IEncoder {
+	//TODO: Find a way to rename this class to Encoder without conflicting
+	//      with wpilib's Encoder class.
```

Comment:
Okay, I had to deal with fully qualified names, and you didn't go with them because they're *TERRIBLE.*